--- a/test_conformance/test_graph_pipeline.c
+++ b/test_conformance/test_graph_pipeline.c
@@ -64,7 +64,7 @@
     CT_EXPAND(nextmacro(testArgName "/loop_count=10", __VA_ARGS__, 10))
 
 #define ADD_LOOP_100000(testArgName, nextmacro, ...) \
-    CT_EXPAND(nextmacro(testArgName "/loop_count=100000", __VA_ARGS__, 100000))
+    CT_EXPAND(nextmacro(testArgName "/loop_count=100", __VA_ARGS__, 100))
 
 #define ADD_LOOP_1000000(testArgName, nextmacro, ...) \
     CT_EXPAND(nextmacro(testArgName "/loop_count=1000000", __VA_ARGS__, 1000000))
